__init__.py

```diff
@@ -110,19 +115,26 @@ def myRevHtml(reviewer, _old):
     paste = bool_to_str(config["process_paste"])
     rem_span = bool_to_str(config["remove_span"])
     special = json.dumps(config["z_special_formatting"])
+    preserve_ratio = config.get("preserve_ratio", "current")
+    min_height = config.get("min_height", 10)
+    min_width = config.get("min_width", 10)
+    style = JQUERYUICSS
+    resizable_style = config.get("resizable_style", "border:1px dashed black;")
+    if resizable_style:
+        style += f"<style>.ui-wrapper {{   {resizable_style} }}</style>"
 
     js = GLOBALCARDJS % ({"span": span, "ctrl": ctrl, "paste": paste,
                           "remove_span": rem_span, "special": special})
+    js += RESIZEJS % ({"min_height": min_height, "min_width": min_width, "preserve_ratio": preserve_ratio, "resizable_style": resizable_style});
 
     if config["process_paste"]:
         js += PASTEJS
 
     if config["outline"]:
-        style = "<style>[data-efdrc='true'][contenteditable='true']:focus{outline: 1px solid #308cc6;}</style>"
-    else:
-        style = ""
+        style += "<style>[data-efdrc='true'][contenteditable='true']:focus{outline: 1px solid #308cc6;}</style>"
 
-    return _old(reviewer) + js + style
+    print(f"style is «{style}»")
```

Comment:
Perhaps you forgot to delete this line? Although I guess users won't know the difference

Reply:
I indeed forgot it. Sorry.
I will clearly see the difference, since I use the terminal a lot to debug. I'll remove that immediately